content/shmem_team_split_2d.tex

```diff
@@ -0,0 +1,111 @@
+void @\FuncDecl{shmem\_team\_split\_2d}@(shmem_team_t parent_team, int xrange,
+long xaxis_options, shmem_team_t *xaxis_team, long yaxis_options, shmem_team_t *yaxis_team);
+\end{Csynopsis}
+
+\begin{apiarguments}
+\apiargument{IN}{parent\_team}{A valid \openshmem team. Any predefined teams, such as
+\LibHandleRef{SHMEM\_TEAM\_WORLD}, may be used, or any team created by the user.}
+
+\apiargument{IN}{xrange}{A non-negative integer representing the number of
+elements in the first dimension.}
+
+\apiargument{IN}{xaxis\_options}{The set of options requested for the new X-axis team.
+Multiple options may be requested by combining them with a bitwise OR operation;
+otherwise, \CONST{0} can be given if no options are requested.}
+
+\apiargument{OUT}{xaxis\_team}{A new \ac{PE} team handle representing a \ac{PE}
+subset consisting of all the \acp{PE} that have the same coordinate along the X-axis 
+as the calling \ac{PE}.}
+
+\apiargument{IN}{yaxis\_options}{The set of options requested for the new Y-axis team.
+These options do not have to be the same as the options for the new X-axis team.}
+
+\apiargument{OUT}{yaxis\_team}{A new \ac{PE} team handle representing a \ac{PE}
+subset consisting of all the \acp{PE} that have the same coordinate along the Y-axis 
+as the calling \ac{PE}.}
+\end{apiarguments}
+
+\apidescription{
+The \FUNC{shmem\_team\_split\_2d} routine is a collective routine. It creates two new teams
+by splitting an existing parent team into up to two subsets based on a
+2D Cartesian space. The user provides the size of the X dimension, which is then used to
+derive the size of the Y dimension based on the size of the parent team. The size of
+the Y dimension will be equal to $ceiling ( N \div xrange )$, where \VAR{N} is the size
+of the parent team. In other words, $xrange \times yrange \geq N$, so that every \ac{PE}
+in the parent team has a unique \VAR{(x,y)} location the 2D Cartesian space.
+
+After the split operation, each of the new teams will contain all \acp{PE} that have the same
+coordinate along the X-axis and Y-axis, respectively, as the calling \ac{PE}. The
+\acp{PE} are numbered in the new teams based on the position of the \ac{PE} along
+the given axis.
+
+Any valid \openshmem team can be used as the parent team. This routine must be
+called by all \acp{PE} in the parent team. The value of \VAR{xrange} must be
+non-negative and all \acp{PE} in the parent team must pass the same value for
+\VAR{xrange}. None of the parameters need to reside in symmetric memory.
+
+Since the split may result in a 2D space with more points than there are members of
+the parent team, there may be a final, incomplete row of the 2D mapping of the parent
+team. This means that the resulting X-axis teams may vary in size by up to 1 \ac{PE},
+and that there may be one resulting Y-axis team of smaller size than all of the other
+Y-axis teams.
+
+The following grid shows the twelve teams that would result from splitting a parent team
```

Comment:
I think that this example (and maybe the following paragraph) should be in the _Notes_ section, not the behavioral description.